src/test/java/org/zalando/problem/spring/web/advice/ExceptionHandling.java

```diff
@@ -20,8 +20,15 @@
  * #L%
  */
 
+import com.google.common.base.CaseFormat;
 import org.springframework.web.bind.annotation.ControllerAdvice;
 
 @ControllerAdvice
 public final class ExceptionHandling implements ProblemHandling {
+
+    @Override
+    public String formatFieldName(String fieldName) {
+        return CaseFormat.LOWER_CAMEL.to(CaseFormat.LOWER_UNDERSCORE, fieldName);
```

Comment:
> Can we make this configurable or somehow access the naming conventions of the objectmapper used for deserializing the request?

I don't think there is a way to access anything related to deserialization at this point. The second problem, even if we would be able to access the `PropertyNamingStrategy` - it only works on annotated elements or fields, nothing we can reliably reproduce at this point.